packages/components/List/index.js

```diff
@@ -0,0 +1,64 @@
+import React, { useState } from 'react';
+import SimpleCard from '../SimpleCard';
+import SearchBar from '../SearchBar';
+
+const List = ({ items, handleSearch, handleEdit, handleDelete }) => {
+
+    const includesText = (textOne, textTwo) => {
+        return textOne.toLowerCase().trim().includes(textTwo.toLowerCase().trim());
+    };
+
+    const [list, handleUpdateList] = useState(items);
+
+    handleSearch = searchText => {
+        const results = items.filter(item => {
+            if (
+                includesText(item.title, searchText) ||
+                includesText(item.description, searchText)
+            ) {
+                item.display = true;
+
+                return item;
+            } else {
+                item.display = false;
+
+                return item;
+            }
+        });
```

Comment:
Suggestion: Might be worth moving the inner function of `items.filter` to be its own function so you could do something more simple like:

```
const results = items.filter(item => shouldDisplay(item, searchTerm));
```